on-chain-metrics.md

```diff
@@ -102,8 +102,3 @@ List of with metrics with 1 day minimal resolution:
 ### NVT
 
 - nvt
-
-List of with metrics with 5 minutes minimal resolution:
-
-- transaction_volume_5min
-- age_destroyed_5min
```

Comment:
`age_destroyed` and `transaction_volume` are available per 5 minute, only not with this `_5min` name

Reply:
Got it